dbhydra/dbhydra_core.py

```diff
@@ -481,17 +485,17 @@ class Mysqldb(AbstractDB):
     'Jsonable': "json"
     }
 
-    def connect_to_db(self):
+    def connect_to_db_globally(self):
+        """Connect to the DB globally (not as a ContextManager)"""
```

Comment:
what about backward compatibility? Breaking change here

Reply:
It's backward-compatible, `connect_to_db` is provided by `AbstractDB` class